nats-streaming-server.go

```diff
@@ -195,6 +198,9 @@ func parseFlags() (*stand.Options, *natsd.Options) {
 	flag.Int64("file_fds_limit", stores.DefaultFileStoreOptions.FileDescriptorsLimit, "FileStoreOpts.FileDescriptorsLimit")
 	flag.Int("io_batch_size", stand.DefaultIOBatchSize, "IOBatchSize")
 	flag.Int64("io_sleep_time", stand.DefaultIOSleepTime, "IOSleepTime")
+	flag.String("ft_group", "", "FTGroupName")
+	flag.Int("ft_quorum", 1, "FTQuorum")
```

Comment:
So long as it is clear to a typical user that this is the minimum # of servers, sounds good here.